### PR TITLE
Homework 4(web functions)

### PR DESCRIPTION
**Pull request by HunkevychPhilip**

### CODE REVIEW

File: src/web/index.php [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+             - when you apply filter_by_state the page should be equal 1
+             - when you apply filter_by_state, than filter_by_first_letter (see Filtering task #1) is not reset
+               i.e. if you have filter_by_first_letter set you can additionally use filter_by_state
+        -->
+
+         <?php foreach ($airports as $airport): ?>
+         <tr>
+             <td><?= $airport['name'] ?></td>
+             <td><?= $airport['code'] ?></td>
+             <td><a href="<?= "/?" . http_build_query(array_merge($_GET, ['page' => 1, 'filter_by_state' => $airport['state']])) ?>"><?= $airport['state'] ?></a></td>
```
Comment (yderevianko-cartera) [inline src/web/index.php]: you can move query generation to separate function

File: tests/GetUniqueFirstLettersTest.php [hunk 2]
```diff
@@ -0,0 +1,32 @@
+<?php
+
+use PHPUnit\Framework\TestCase;
+
+class GetUniqueFirstLettersTest extends TestCase
+{
+    /**
+     * @dataProvider providerPositiveData
+     */
+    public function testPositive($input, $expected)
```
Comment (yderevianko-cartera) [inline tests/GetUniqueFirstLettersTest.php:10]: please add more test for all functions and provide some negative test cases if possible

**Review by yderevianko-cartera — CHANGES_REQUESTED** [review]

**Review by yderevianko-cartera — APPROVED** [review]
